readthedocs/templates/errors/404/include_search.html

```diff
@@ -0,0 +1,13 @@
+{% load i18n %}
+
+<h2>{% trans "Try searching?" %}</h2>
```

Comment:
This is including two titles

![Screenshot 2023-04-19 at 11-59-19 404 Not Found Read the Docs](https://user-images.githubusercontent.com/4975310/233147353-1212aa10-57a2-40a3-b4e8-4ed141ee4fa0.png)

I think we should remove the "search the docs"

Reply:
Fixed